### ISSUE TITLE
Gutenberg: Embed Blocks: Unexpected behavior in various Embed Blocks

### ISSUE THREAD

**Issue by JavonDavis** [Automattic/wp-calypso#28126]
#### Steps to reproduce
1. Starting at URL: https://wpcalypso.wordpress.com/gutenberg/post
2.Select any site Simple or Jetpack
3. Create a new Block from under embed
4. Enter appropriate URL

#### What I expected
* Expected nothing to happen really until I click embed or maybe a preview before

#### What happened instead

An error message stating "Sorry, we could not embed that content." appeared.

#### Browser / OS version

Chrome v70
Gutenberg Version 4.0.0

#### Screenshot / Video
<img width="1642" alt="screen shot 2018-10-26 at 2 18 48 pm" src="https://user-images.githubusercontent.com/6520461/47589827-f6aca500-d92f-11e8-960c-fa4fe5f67551.png">

<img width="1665" alt="screen shot 2018-10-26 at 2 19 02 pm" src="https://user-images.githubusercontent.com/6520461/47589826-f6aca500-d92f-11e8-9950-19c1b55cb552.png">

**Comment by correliebre** [comment]
Wanted to add that I tried it with different embeds and got different results:

**1. Instagram link:** 

Error saying it can't be embed. 

![Instagram](https://cdn-std.dprcdn.net/files/acc_652292/36UPWD)

Front-end shows it correct, though: 

![Instagram Front-End](https://cdn-std.dprcdn.net/files/acc_652292/M4qU2s)

**2. Spotify link:**

No error:

![Spotify](https://cdn-std.dprcdn.net/files/acc_652292/NFjHOZ)

**3. Facebook link:**

Error too big to be displayed in the correct area.

![Facebook](https://cdn-std.dprcdn.net/files/acc_652292/oCkRi7)

Front-end shows it correct, though:

![Facebook Front-End](https://cdn-std.dprcdn.net/files/acc_652292/Cc7oNd)

**4. Twitter link:**

No error:

![Twitter](https://cdn-std.dprcdn.net/files/acc_652292/WHOKc1)

**Desktop:**

    OS: Mojave 10.14
    Browser: Firefox
    Version: 62.0.3

**Additional context:**

    Atomic site with only Gutenberg installed, besides Jetpack

**Comment by JavonDavis** [comment]
Status update on all Embed Blocks

TLDR;

**Working completely as Expected**

YouTube
Twitter
Dailymotion
Flickr
Imgur
Issuu
Polldaddy
Reddit

**Not working as expected(Problem):** CNE - Could not embed

Embed (Not sure what other URL works here)
WordPress(Publish Only)
Facebook(Both)
Instagram(Both) 
Spotify(CNE)
Soundcloud(Editor Only)
Vimeo(Both)
Animoto(Editor Only)
Cloudup(Editor Only)
College Humor(Both)
Kickstarter(Both)
Meetup(CNE)
Mixcloud(CNE)
Photobucket(CNE)
ReverbNation(CNE)
Scribd(CNE)
Slideshare(Editor Only)
SmugMug(Editor Only)
Speaker Deck(Console Error)
Ted(Editor Only)
Tumblr(Publish Only)
VideoPress(Editor Only)
WordPress.tv(Editor Only) 


Noticed on a poor internet connection it blanks out like this for embeds in the editor

<img width="1519" alt="unknown" src="https://user-images.githubusercontent.com/6520461/47754476-00ecdd00-dc69-11e8-979a-86f718e157c1.png">

And the publish preview is empty

Also noticed this weird behavior sometimes


Editor Preview

![unknown-1](https://user-images.githubusercontent.com/6520461/47754470-fdf1ec80-dc68-11e8-99c1-b26f565e95f7.png)

Publish Preview

![unknown-2](https://user-images.githubusercontent.com/6520461/47754548-34c80280-dc69-11e8-884d-5439c2323c35.png)


**Youtube**

Tested with URL: https://www.youtube.com/watch?v=5dsGWM5XGdg
Editor Preview: 
![unknown-3](https://user-images.githubusercontent.com/6520461/47754557-3bef1080-dc69-11e8-9afd-bbf252e9d364.png)

Published Preview:
![unknown-4](https://user-images.githubusercontent.com/6520461/47754560-3e516a80-dc69-11e8-8ec1-28ff322039c1.png)



**Twitter**

Tested with URL: https://twitter.com/automattic
Editor Preview:

![unknown-5](https://user-images.githubusercontent.com/6520461/47754677-af911d80-dc69-11e8-9af3-76c98d56bd7d.png)

Published Preview: 

![unknown-6](https://user-images.githubusercontent.com/6520461/47754698-be77d000-dc69-11e8-9723-7b3cdeb5c843.png)


**Embed**

Tested with URL: https://en.wikipedia.org/wiki/Cat
Editor Preview: “Sorry, we could not embed that content.“
Published Preview: N/A

Couldn’t figure out a URL that could work here besides one valid for another more specific type of embed

**WordPress**

Tested with URL: https://testing201814.blog/
Tested with URL: https://testing201814.wordpress.com
Tested with URL: https://testing201814.blog/2018/10/23/

Editor Preview: “Sorry, we could not embed that content.“
Published Preview: N/A

Tested with URL: https://testing201814.blog/2018/10/23/gutenberg-tiled-gallery-block/

Editor Preview:
<img width="626" alt="unknown-7" src="https://user-images.githubusercontent.com/6520461/47754711-cfc0dc80-dc69-11e8-9421-37401f278c6c.png">


Published Preview:
<img width="546" alt="unknown-8" src="https://user-images.githubusercontent.com/6520461/47754713-d2233680-dc69-11e8-881a-b5248c4c2f7f.png">



This seems like it’s intentional but I expected embedded content similar to the Editor Preview from at least https://testing201814.blog/2018/10/23/

**Dailymotion**

Tested with URL: https://www.dailymotion.com/video/x6w7aph?playlist=x63q3a
Editor Preview: 
<img width="660" alt="unknown-9" src="https://user-images.githubusercontent.com/6520461/47754724-da7b7180-dc69-11e8-8386-d80e90497a9d.png">

Published Preview: 

<img width="548" alt="unknown-10" src="https://user-images.githubusercontent.com/6520461/47754728-de0ef880-dc69-11e8-80c3-6f2c182dce92.png">


**Facebook:**

Tested with URL: https://web.facebook.com/WordPress/
Tested with URL: https://web.facebook.com/WordPress/posts/10156301634992911?__tn__=-R

Editor Preview: 
<img width="1519" alt="unknown-11" src="https://user-images.githubusercontent.com/6520461/47754772-03036b80-dc6a-11e8-97ca-43787f23a933.png">


Published Preview: 
<img width="561" alt="unknown-12" src="https://user-images.githubusercontent.com/6520461/47754779-08f94c80-dc6a-11e8-8c83-fc07e86be582.png">


**Instagram:**

Tested with URL: https://www.instagram.com/_codingblog/
Tested with URL: https://www.instagram.com/p/BpiEuYRhsev/?utm_source=ig_web_button_share_sheet

Editor Preview

<img width="1369" alt="unknown-13" src="https://user-images.githubusercontent.com/6520461/47754787-0bf43d00-dc6a-11e8-8372-ec684e3eb76c.png">



Publish Preview: N/A



**Spotify:**

Tested with URL: https://open.spotify.com/playlist/37i9dQZF1DX2A29LI7xHn1
Tested with URL: https://open.spotify.com/search/results/drake

Editor Preview:

![unknown-14](https://user-images.githubusercontent.com/6520461/47754794-0e569700-dc6a-11e8-8658-8c7da9cf45bb.png)

Publish Preview: N/A

**Soundcloud:**

Tested with URL: https://soundcloud.com/lil_peep/lil-peep-ft-xxxtentacion-falling-down

Editor Preview:
![unknown-15](https://user-images.githubusercontent.com/6520461/47754798-10b8f100-dc6a-11e8-8d45-31c76c953e25.png)

Publish Preview: 

![unknown-16](https://user-images.githubusercontent.com/6520461/47754833-28907500-dc6a-11e8-981e-fc17d9ab9091.png)



**Flickr:**

Test with URL: https://flic.kr/p/2cpgoHs

Editor Preview:

![unknown-17](https://user-images.githubusercontent.com/6520461/47754839-2d552900-dc6a-11e8-84ca-43641278a685.png)

Publish Preview:

<img width="1261" alt="unknown-18" src="https://user-images.githubusercontent.com/6520461/47754844-3219dd00-dc6a-11e8-9776-4a5f120be3ee.png">


**Vimeo:**

Tested with URL: https://vimeo.com/channels/staffpicks/295354277

Editor Preview: 
![unknown-19](https://user-images.githubusercontent.com/6520461/47754854-380fbe00-dc6a-11e8-9ca3-c2112784a331.png)

Publish preview: 

![unknown-20](https://user-images.githubusercontent.com/6520461/47754855-38a85480-dc6a-11e8-8b59-cd47e3e09bb7.png)


**Animoto:**

Tested with URL: https://animoto.com/play/WmGs0SgMeHvBMur0fL68rw

Editor Preview:

![unknown-21](https://user-images.githubusercontent.com/6520461/47754856-3b0aae80-dc6a-11e8-89a3-880cc4c2a708.png)


Publish Preview:
![unknown-22](https://user-images.githubusercontent.com/6520461/47754865-3f36cc00-dc6a-11e8-8c89-6d803239afea.png)


**Cloudup:**

Tested with URL: https://cloudup.com/if61tFHyBgA

Editor Preview

![unknown-23](https://user-images.githubusercontent.com/6520461/47754890-5aa1d700-dc6a-11e8-92c7-dbc60688b189.png)


Publish Preview
![unknown-24](https://user-images.githubusercontent.com/6520461/47754893-5d043100-dc6a-11e8-9a5f-9f0a02f57330.png)



**College Humor**

Tested with URL: http://www.collegehumor.com/video/40004082/the-girl-with-a-birthday-month

Editor Preview:

<img width="1341" alt="unknown-25" src="https://user-images.githubusercontent.com/6520461/47754898-5f668b00-dc6a-11e8-80de-32d000736822.png">


Publish Preview:
<img width="1522" alt="unknown-26" src="https://user-images.githubusercontent.com/6520461/47754901-62617b80-dc6a-11e8-8e9d-af4d6cbfaa3e.png">


I think this one specifically fails due to the URL being not over https and suspect this would apply to any http embed URL.

Console log:

`Mixed Content: The page at 'https://jetpack-gutenberg-testing.mystagingwebsite.com/?p=44&preview=true' was loaded over HTTPS, but requested an insecure resource 'http://www.collegehumor.com/e/40004082'. This request has been blocked; the content must be served over HTTPS.`

**Funny Or Die**

Tested with URL: https://www.funnyordie.com/best-of-the-web
Tested with URL: https://www.funnyordie.com/best-of-the-web/2018/10/29/18040122/big-asian-naturals-supermarket-vice-parody-documentary-jamaal-sedayao
Tested with URL: https://www.funnyordie.com/2018/10/30/18040648/dumb-spanish-dubbing-halloween-silence-of-the-lambs-childs-play-it

Editor Preview: “Sorry, we could not embed that content.“
Published Preview: N/A

**Imgur**

Tested with URL: https://imgur.com/gallery/v7Qs8ND


Editor Preview:
<img width="1354" alt="unknown-27" src="https://user-images.githubusercontent.com/6520461/47754926-70170100-dc6a-11e8-8355-e63ebb2b0fbd.png">


Published Preview:

<img width="1546" alt="unknown-28" src="https://user-images.githubusercontent.com/6520461/47754933-72795b00-dc6a-11e8-98f4-cbe6a3f18c1f.png">


**Issuu**

Tested with URL: https://issuu.com/mymagazinebymollymy/docs/my_magazine_fall2017_finaloct18/34

Editor Preview:

<img width="1367" alt="unknown-29" src="https://user-images.githubusercontent.com/6520461/47754937-77d6a580-dc6a-11e8-981e-52b8b8ed4712.png">

Published Preview:

<img width="1490" alt="unknown-30" src="https://user-images.githubusercontent.com/6520461/47754940-7b6a2c80-dc6a-11e8-938b-aa82408d3eae.png">


**Kickstarter**

Tested with URL: https://www.kickstarter.com/projects/1683809116/manjits-kitchen-and-bar?ref=discovery

Editor Preview: 
<img width="1362" alt="unknown-31" src="https://user-images.githubusercontent.com/6520461/47754945-7dcc8680-dc6a-11e8-98c6-e7c98c89a0ea.png">

Published Preview:
<img width="1170" alt="unknown-32" src="https://user-images.githubusercontent.com/6520461/47754949-802ee080-dc6a-11e8-9c48-dd980c36b649.png">


**Meetup.com**

Tested with URL: https://www.meetup.com/Kingston-WordPress-Meetup/events/smvthpyxpblb/

Editor Preview: “Sorry, we could not embed that content.“
Published Preview: N/A

**Mixcloud**

Tested with URL: https://www.mixcloud.com/melodicdistraction/born-to-roll-1st-birthday-special-october-18/

Editor Preview: “Sorry, we could not embed that content.“
Published Preview: N/A

**Photobucket**

Tested with URL: http://photobucket.com/gallery/user/bstevens2002/media/bWVkaWFJZDoxNjQ4MTUzNDA=/?ref=

Editor Preview: “Sorry, we could not embed that content.“
Published Preview: N/A

**Polldaddy (Now Crowdsignal)**


Tested with URL: https://poll.fm/10151656

Editor Preview: 
<img width="1324" alt="unknown-33" src="https://user-images.githubusercontent.com/6520461/47754956-8755ee80-dc6a-11e8-850d-716885d1cb71.png">

Published Preview: 
<img width="1140" alt="unknown-34" src="https://user-images.githubusercontent.com/6520461/47754957-89b84880-dc6a-11e8-84c8-3e92b77fc4e3.png">


**Reddit**

Tested with URL: https://www.reddit.com/r/webdev

Editor Preview: “Sorry, we could not embed that content.“
Published Preview: N/A

Tested with URL: https://www.reddit.com/r/webdev/comments/9rkm4f/beginner_questions_october_26_2018/

Editor Preview:
<img width="1337" alt="unknown-35" src="https://user-images.githubusercontent.com/6520461/47754965-8fae2980-dc6a-11e8-9c6d-7ae3c2e7a318.png">

Published Preview: 
<img width="1280" alt="unknown-36" src="https://user-images.githubusercontent.com/6520461/47754968-92108380-dc6a-11e8-8cd7-4816a3cd09bf.png">


**ReverbNation**

Tested with URL: https://www.reverbnation.com/collection/186-mellow-beats

Editor Preview: “Sorry, we could not embed that content.“
Published Preview: N/A

**Screencast**

Did not test

**Scribd**

Site not working atm….
<img width="2036" alt="unknown-37" src="https://user-images.githubusercontent.com/6520461/47755012-bf5d3180-dc6a-11e8-9155-3f1e525c1439.png">


Tested with this URL: https://www.scribd.com/article/391724241/

Editor Preview: “Sorry, we could not embed that content.“
Published Preview: N/A


**Slideshare**

Tested with URL: https://www.slideshare.net/reidhoffman/blitzscaling-book-trailer-118631898

Editor Preview: 
<img width="1350" alt="unknown-38" src="https://user-images.githubusercontent.com/6520461/47755022-c8e69980-dc6a-11e8-85cb-634b65723b87.png">

Published Preview: 
<img width="1534" alt="unknown-39" src="https://user-images.githubusercontent.com/6520461/47755025-cb48f380-dc6a-11e8-9e48-a73c59467628.png">


**SmugMug**

Tested with URL: https://nelli.smugmug.com/Daily-photos/i-bXKH8bz

Editor Preview: 
<img width="1320" alt="unknown-40" src="https://user-images.githubusercontent.com/6520461/47755037-d6038880-dc6a-11e8-8df9-62c0455743b0.png">

Published Preview: 
<img width="1135" alt="unknown-41" src="https://user-images.githubusercontent.com/6520461/47755040-d8fe7900-dc6a-11e8-9e74-35adeec2a110.png">


**Speaker Deck**

Tested with URL: https://speakerdeck.com/maltzj/code-reviewing-like-a-champion

Editor Preview:
<img width="1655" alt="unknown-42" src="https://user-images.githubusercontent.com/6520461/47755045-dd2a9680-dc6a-11e8-8ec5-4c4fe1b3cad9.png">

Published Preview:
<img width="739" alt="unknown-43" src="https://user-images.githubusercontent.com/6520461/47755047-df8cf080-dc6a-11e8-8c69-ea12e2be8718.png">


**TED**

Tested with URL: https://www.ted.com/talks/ken_robinson_says_schools_kill_creativity?referrer=playlist-the_most_popular_talks_of_all

Editor Preview: 
<img width="834" alt="unknown-44" src="https://user-images.githubusercontent.com/6520461/47755052-e287e100-dc6a-11e8-92f3-b03c908c1c70.png">

Published Preview:
<img width="1389" alt="unknown-45" src="https://user-images.githubusercontent.com/6520461/47755055-e451a480-dc6a-11e8-8487-248d673fa034.png">



**Tumblr**

Tested with URL: http://iglovequotes.net/post/179598785847/httpiglovequotesnet

Editor Preview: 
<img width="1663" alt="unknown-46" src="https://user-images.githubusercontent.com/6520461/47755061-e9165880-dc6a-11e8-8c36-c37a2e668d10.png">

Published Preview:
<img width="1483" alt="unknown-47" src="https://user-images.githubusercontent.com/6520461/47755062-eb78b280-dc6a-11e8-81c0-89691aeadd6d.png">


**VideoPress**

Tested with URL: https://videopress.com/v/kUJmAcSf

Editor Preview:
<img width="1650" alt="unknown-48" src="https://user-images.githubusercontent.com/6520461/47755067-efa4d000-dc6a-11e8-81b4-97f05b4c94d1.png">

Published Preview: 
<img width="1337" alt="unknown-49" src="https://user-images.githubusercontent.com/6520461/47755069-f29fc080-dc6a-11e8-85ab-684e2c884687.png">


**WordPress.tv**

Tested with URL: https://wordpress.tv/2018/07/04/matt-mullenweg-a-summertime-update-keynote-and-qa/

Editor Preview: 
<img width="1662" alt="unknown-50" src="https://user-images.githubusercontent.com/6520461/47755073-f59ab100-dc6a-11e8-946c-709cd7d39eed.png">

Published Preview:
<img width="1326" alt="unknown-51" src="https://user-images.githubusercontent.com/6520461/47755075-f7fd0b00-dc6a-11e8-903f-cb1629c13f84.png">

**Comment by JavonDavis** [comment]
FYI youtube Embed not working in wpcalypso, Related -> https://github.com/Automattic/wp-e2e-tests/pull/1670

**Comment by cecoates** [comment]
Also added here:

https://github.com/Automattic/wp-calypso/issues/27708#issuecomment-445372797

But I have a user whose Getty images appear to all be broken by the recent update. And the only way to fix it is by manually updated each specific Getty embed.

**Comment by gwwar** [comment]
Can we please open an issue per broken embed? For easier viewing I've also created a milestone to add this to. Reports are very much welcome if you're unable to find existing issues.

https://github.com/Automattic/wp-calypso/milestone/255

**Comment by JavonDavis** [comment]
> Can we please open an issue per broken embed? For easier viewing I've also created a milestone to add this to. Reports are very much welcome if you're unable to find existing issues.

@gwwar sure, I'll work on this... We also still have -> https://gutenpackembeds.mystagingwebsite.com that has all the blocks, I'll try a put this somewhere for reference
